assets/js/students.json

```diff
@@ -0,0 +1 @@
+[{"name":"Student Name"}]
```

Comment:
Have the students submit a PR to add their own data (this will probably create merge conflict hell and we may need some clever way around that)